projects/packages/wpcom-posts/actions.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
Maybe this is something that should be within the Jetpack plugin instead, or within wpcomsh, and then this package would only be the package itself?

Reply:
I'd prefer to leave this out of the Jetpack plugin for various reasons:
- This is part of a technical spike exploring how we can build a package which will be exclusively consumed by WP.com Simple and Atomic sites.
- I don't see a clear fit for this feature in Jetpack (especially after branding it as a solution for Security, Backups, Performance, and Growth). Even if we decide there is a fit, there is not certainty that all future features that will be bundled in this package may fit.

I do see value in setting up this via the Config package (which will be done on the WP.com sites), so I'll explore that approach.